src/utils.ts

```diff
@@ -87,7 +87,7 @@ export const generateCommitMessage = async (
 			n: 1,
 		});
 
-		return completion.data.choices[0].text!.trim().replace(/[\n\r]/g, '');
+		return completion.data.choices[0].text!.trim().replace(/[\n\r]/g, '').replace(/\.$/, "");
```

Comment:
Don't want to catch `...` so let's do something like:

```suggestion
		return completion.data.choices[0].text!.trim().replace(/[\n\r]/g, '').replace(/(\w)\.$/, '$1');
```

```js
// Doesn't strip period
'asdf...'.replace(/(\w)\.$/, '$1') => 'asdf...'

// Strips last period
'asdf.'.replace(/(\w)\.$/, '$1') => 'asdf'
```